src/model/UserModel.ts

```diff
@@ -1,50 +1,50 @@
-import { Model, DataTypes } from "sequelize"
-import { sequelize } from "../db/config"
+import { Model, DataTypes } from "sequelize";
+import { sequelize } from "../db/config";
 
 class UserModel extends Model {
-    public id!: number
-    public firstName!: string
-    public lastName!: string
-    public email!: string
-    public password!: string
-    public role!: string
-    public readonly createdAt!: Date
-    public readonly updatedAt!: Date
+  public id!: number;
+  public firstName!: string;
+  public lastName!: string;
+  public email!: string;
+  public password!: string;
+  public role!: string;
+  public readonly createdAt!: Date;
```

Comment:
Can we please have one file, because as you can see we have UserModel.ts and user.model.ts

Reply:
Yeah, you are right. I'll remove it.